README.md

```diff
@@ -34,4 +34,4 @@ This data in addition to the other single-cell metadata is stored as a parquet f
 To understand each treatments ability to induce each of the 15 phenotypes, we perform post hoc testing with the probabilities between each of the treatments and the control (DMSO).
 These probabilities were generated in `2.evaluate_data`.
 In each of these tests we compute the median to determine the directionality of significance among treatments.
```

Comment:
Should you explain why you use both Dunn and Mann Whitney tests?

Or should this be included in a README for the `3.analyze_data` module?